cypress/integration/testing.js

```diff
@@ -0,0 +1,129 @@
+describe('rancid-tomatillos movie display app', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+      statusCode: 200,
+      body: {
+        movies: [
+          {
+            id: 694919,
+            poster_path: "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
+            backdrop_path: "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
+            title: "Money Plane",
+            average_rating: 6.875,
+            release_date: "2020-09-29"
+            },
+            {
+            id: 337401,
+            poster_path: "https://image.tmdb.org/t/p/original//aKx1ARwG55zZ0GpRvU2WrGrCG9o.jpg",
+            backdrop_path: "https://image.tmdb.org/t/p/original//zzWGRw277MNoCs3zhyG3YmYQsXv.jpg",
+            title: "Mulan",
+            average_rating: 5.1,
+            release_date: "2020-09-04"
+            },
+            {
+            id: 718444,
+            poster_path: "https://image.tmdb.org/t/p/original//uOw5JD8IlD546feZ6oxbIjvN66P.jpg",
+            backdrop_path: "https://image.tmdb.org/t/p/original//x4UkhIQuHIJyeeOTdcbZ3t3gBSa.jpg",
+            title: "Rogue",
+            average_rating: 7.333333333333333,
+            release_date: "2020-08-20"
+            },
+            {
+            id: 539885,
+            poster_path: "https://image.tmdb.org/t/p/original//qzA87Wf4jo1h8JMk9GilyIYvwsA.jpg",
+            backdrop_path: "https://image.tmdb.org/t/p/original//54yOImQgj8i85u9hxxnaIQBRUuo.jpg",
+            title: "Ava",
+            average_rating: 5.875,
+            release_date: "2020-07-02"
+            },
+            {
+            id: 581392,
+            poster_path: "https://image.tmdb.org/t/p/original//sy6DvAu72kjoseZEjocnm2ZZ09i.jpg",
+            backdrop_path: "https://image.tmdb.org/t/p/original//gEjNlhZhyHeto6Fy5wWy5Uk3A9D.jpg",
+            title: "Peninsula",
+            average_rating: 7.166666666666667,
+            release_date: "2020-07-15"
+            }
+        ]
+      }
+    })
+    cy.visit('http://localhost:3000')
+  })
+
+  it('User should see the Rancid Tomatillos navbar', () => {
+    cy.get('nav').contains('Rancid Tomatillos')
+  })
+
+  // Replace this with Search when we add that feature!
+
+  it('User should see Pick A Movie in the navbar', () => {
+    cy.get('nav').contains('Pick A Movie!')
+  })
+
+  it('User should see 5 movie previews at the homepage', () => {
+    cy.get('.movie-container').contains('Money Plane')
```

Comment:
Glad we're testing all, wonder if we need to test that the images are showing up too eventually--but great for now!